tests/unit/source_code/test_dependencies.py

```diff
@@ -147,27 +128,6 @@ def test_dependency_resolver_raises_problem_with_unresolved_import(mock_path_loo
     ]
 
 
-def test_dependency_resolver_raises_problem_with_non_constant_notebook_argument(mock_path_lookup):
-    notebook_loader = NotebookLoader()
-    notebook_resolver = NotebookResolver(notebook_loader)
-    whitelist = Whitelist()
-    import_resolver = ImportFileResolver(FileLoader(), whitelist)
-    pip_resolver = PythonLibraryResolver(whitelist)
-    dependency_resolver = DependencyResolver(pip_resolver, notebook_resolver, import_resolver, mock_path_lookup)
-    maybe = dependency_resolver.build_local_file_dependency_graph(Path("run_notebooks.py"))
-    assert list(maybe.problems) == [
-        DependencyProblem(
-            'dependency-not-constant',
-            "Can't check dependency not provided as a constant",
-            Path("run_notebooks.py"),
-            14,
-            13,
-            14,
-            50,
-        )
-    ]
-
```

Comment:
Same comment about this test: should this be updated instead of being removed? 🤔

Reply:
We already have a test for checking value inference. I don't think it's reasonable to keep this test. The checked Advice no longer exists

Reply:
These @ tests were more about checking that the generated Advice made it upwards to the caller